src/components/NewLocationPage/ChartFooter/MetricChartFooter.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import {
+  Footer,
+  ButtonContainer,
+  FooterText,
+  AboutText,
+  ModalButton,
+  DisclaimerWrapper,
+} from './ChartFooter.style';
+import NewDialog from 'components/NewDialog/NewDialog';
+import { Link } from 'cms-content/modals';
+import { MobileOnly, DesktopOnly } from '../Shared/Shared.style';
+import ShareButtons from 'components/LocationPage/ShareButtons';
+import { Region } from 'common/regions';
+import type { MetricValues } from 'common/models/Projections';
+
+export interface AboutModalProps {
+  header: string;
+  body: string;
+  links: Link[];
+}
+
+export interface ShareButtonProps {
+  region: Region;
+  stats: MetricValues;
+  chartIdentifier: number;
+  showEmbedButton?: boolean;
+}
+
+const MetricChartFooter: React.FC<{
+  footerText: string;
+  disclaimer?: string;
+  shareButtonProps: ShareButtonProps;
+  aboutModal: AboutModalProps;
+}> = ({ footerText, disclaimer, shareButtonProps, aboutModal }) => {
+  const [openModal, setOpenModal] = useState(false);
+  const dialogProps = {
+    open: openModal,
+    closeDialog: () => setOpenModal(false),
```

Comment:
you can use [the useDialog hook](https://github.com/covid-projections/covid-projections/blob/develop/src/common/hooks/useDialog.ts) for some of these props. you can see [how its used here](https://github.com/covid-projections/covid-projections/blob/develop/src/components/Dialogs/MetricInfoDialog.stories.tsx#L26)